### ISSUE TITLE
Error deploying immudb on EKS cluster with EBS added

### ISSUE THREAD

**Issue by MathieuCesbron** [codenotary/immudb#1920]
**What happened**

Hello, I deployed a immudb statefulset on EKS cluster. To add storage I added [Amazon EBS CSI Driver](https://eu-west-3.console.aws.amazon.com/eks/home?region=eu-west-3#/clusters/immudb-operator-example-3/add-ons/aws-ebs-csi-driver). The persistent volume is created but I get an error in the immudb container:

<img width="1238" alt="Screenshot 2024-02-07 at 11 58 45" src="https://github.com/codenotary/immudb/assets/45229023/730fe981-3fc2-4840-b92b-f2c12391a96f">

Basically this error: `ERROR: Unable to get or set uuid: open /var/lib/immudb/immudb.identifier: permission denied`.

Is there something I should add in the manifest ?

**What you expected to happen**
Not having this error

Thanks,

**Comment by MathieuCesbron** [comment]
I just realised I should add to my immudb pod:
```yaml
podSecurityContext:
  runAsNonRoot: true
  runAsUser: 3322
  runAsGroup: 3322
  fsGroup: 3322
  fsGroupChangePolicy: "OnRootMismatch"
```
